src/helpers/functions.js

```diff
@@ -1,5 +1,13 @@
 import { orderTypes } from './consts';
 
+import TomHolland from '../images/people/Tom-Holland.jpeg';
```

Comment:
https://www.delftstack.com/howto/react/import-image-react/ , what will you do when you will have 100 photo?